package.json

```diff
@@ -4,13 +4,12 @@
   "description": "Isomorphic React boilerplate",
   "main": "index.js",
   "scripts": {
-    "start": "node index",
+    "start": "APP_ENV=development node index",
```

Comment:
My issue with this is that if we want to test the production API, we'll have to update the script to be `"start": "APP_ENV=production node index"` and remember to switch the script instead of writing the environment variable when you initially start it up.